mne/io/array/tests/test_array.py

```diff
@@ -139,7 +139,8 @@ def test_array_raw():
 
     # plotting
     raw2.plot()
-    raw2.plot_psd(tmax=2., average=True, n_fft=1024, spatial_colors=False)
+    raw2.plot_psd(picks=['meg','eeg'], tmax=2., average=True, n_fft=1024,
```

Comment:
why did you have to change this?

Reply:
I was getting an Error during testing, but it was my bad, I revoked this change !